aerogear/mobile-developer-console
# 🎨 make sure the data associated with an app is removed when the app is

deleted

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8141

## What

Remove all the data associated with an app when it's been deleted.

## Why

see jira

## How

Add label to bindings, build configs and secrets that are created for an app. When the app is delete, use the label to find the data that should be removed with the app


## Verification Steps

1. Provision some services and bind an app to the services.
2. Make sure you can see the bindings in the OpenShift web console
3. Remove the app
4. You should see the bindings are removed as well in OpenShift web console.

## Checklist:
